test/passing/comments_in_record.ml

```diff
@@ -24,3 +24,8 @@ let { (* cmts *)
     ; (* e *)
       e : loooooooooooooooooooooooooooooooooooooooooooooooooooooooong_int } =
   exp
+
+let x =
+  { (* Xxxx xxxxxxxx xxxxx xx xx xx xxxx xxxxxx - XXxx_xxxxx xxx'x. *)
+  Irure_sed_a.in_nisi_sed= Irure_sed_fugiat.LaboRum sint_sed
```

Comment:
This shows a change that isn't a regression in this patch, it's just different than before, but still not consistent with the behavior for the case where the field is not module-qualified.

Reply:
fix